src/tree_data_internal.h

```diff
@@ -317,6 +428,57 @@ LY_ERR lyd_parse_xml_notif(const struct ly_ctx *ctx, const char *data, struct ly
  */
 LY_ERR lyd_parse_xml_reply(const struct lyd_node *request, const char *data, struct lyd_node **tree, struct lyd_node **op);
 
+/**
+ * @brief Parse binary data as YANG data tree.
+ *
+ * @param[in] ctx libyang context
+ * @param[in] data Pointer to the input data to parse.
+ * @param[in] options @ref dataparseroptions
+ * @param[out] tree Parsed data tree. Note that NULL can be a valid result.
+ * @param[out] parsed_p Optional number of parsed bytes.
```

Comment:
the `_p` suffix seems confusing to me, what about `parsed_bytes` or just `bytes`? The same applies to other functions with this parameter.

Reply:
Renamed.